sql-plugin/src/main/scala/com/nvidia/spark/rapids/ParquetCachedBatchSerializer.scala

```diff
@@ -645,7 +644,7 @@ protected class ParquetCachedBatchSerializer extends GpuCachedBatchSerializer {
         recordReader = columnIO.getRecordReader(pages, new ParquetRecordMaterializer(parquetSchema,
           cacheAttributes.toStructType,
           new ParquetToSparkSchemaConverter(hadoopConf), None /*convertTz*/ ,
-          LegacyBehaviorPolicy.CORRECTED))
+          "CORRECTED"))
```

Comment:
Since we are shimming anyways we could have generated constants instead of hard-codings like `"CORRECTED"`

```Scala
object LegacyBehaviorPolicyShim {
  val CORRECTED_STR: String =  LegacyBehaviorPolicy.CORRECTED.toString
  ...
}
```